### PR TITLE
Add migration file for #4982

### LINKED ISSUE #5023: Database not migrated after addition of location model
**Describe the bug**

Database not migrated after addition of location model in #4982

### PR DESCRIPTION
**Pull request by schedutron**
Fixes #5023

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
This PR adds the missing migration file for PR #4982.

### CODE REVIEW

File: migrations/versions/000b9376b87b_.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    op.create_table('event_locations',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('slug', sa.String(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('slug')
+    )
+    op.add_column('session_types', sa.Column('deleted_at', sa.DateTime(timezone=True), nullable=True))
+    op.add_column('social_links', sa.Column('deleted_at', sa.DateTime(timezone=True), nullable=True))
+    op.add_column('stripe_authorizations', sa.Column('deleted_at', sa.DateTime(timezone=True), nullable=True))
```
Comment (srv-twry) [inline migrations/versions/000b9376b87b_.py]: This shouldn't be here.

Comment (Kreijstal) [reply migrations/versions/000b9376b87b_.py]: Isn't it generated automatically? What do you mean?

Comment (schedutron) [reply migrations/versions/000b9376b87b_.py]: Yeah, there are other changes as well.

Comment (Kreijstal) [reply migrations/versions/000b9376b87b_.py]: Maybe someone forgot to add these columns to the migration or maybe schedutron is not up to date with the migrations

**Comment by srv-twry** [comment]
@schedutron Please check this: https://github.com/fossasia/open-event-server/pull/4960/files

The deleted_at columns were added in this PR. Remove them from your migrations files.

File: migrations/versions/000b9376b87b_.py (hunk 1 above)
Comment (schedutron) [reply migrations/versions/000b9376b87b_.py]: @srv-twry @Kreijstal Were these `deleted_at` fields added in some other migration file?

**Comment by schedutron** [comment]
@srv-twry And the `unique` constraint on `ticket_holders`?

**Comment by srv-twry** [comment]
https://github.com/fossasia/open-event-server/pull/4986/files

From here.

**Comment by schedutron** [comment]
@srv-twry Weird. Only one change from that migration file was catched.

**Comment by bhaveshAn** [comment]
@schedutron 
I guess only `id` `name` & `slug` is required to be migrated.
https://github.com/fossasia/open-event-server/blob/development/app/models/event_location.py

**Pull request merged**